### ISSUE TITLE
--skip-tags is not applied to files in subdirectories

### ISSUE THREAD

**Issue by Octogonapus** [bridgecrewio/yor#296]
**Describe the bug**

Tags that should not be applied due to the `--skip-tags` configuration are applied in files contained within subdirectories relative to where `yor` is run.

```
+---------------+----------------------------------------+----------------------+--------------------------------------+--------------------------------------+
|     FILE      |                RESOURCE                |       TAG KEY        |              TAG VALUE               |                YOR ID                |
+---------------+----------------------------------------+----------------------+--------------------------------------+--------------------------------------+
| events.tf     | xxxxxxxxxxxxxxxxxxxxxxxxxx             | yor_trace            | xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx | xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx |
+               +----------------------------------------+----------------------+--------------------------------------+--------------------------------------+
|               | xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx       | yor_trace            | xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx | xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx |
+               +----------------------------------------+----------------------+--------------------------------------+--------------------------------------+
|               | xxxxxxxxxxxxxxxxxxx                    | yor_trace            | xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx | xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx |
+               +----------------------------------------+----------------------+--------------------------------------+--------------------------------------+
|               | xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx | yor_trace            | xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx | xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx |
+---------------+----------------------------------------+----------------------+--------------------------------------+--------------------------------------+
| smoke/main.tf | xxx                                    | yor_trace            | xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx | xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx |
+               +                                        +----------------------+--------------------------------------+                                      +
|               |                                        | git_repo             | xxxxxxxxxxxxxxxxxxxxxxxx             |                                      |
+               +                                        +----------------------+--------------------------------------+                                      +
|               |                                        | git_org              | xxxxxxxxxxx                          |                                      |
+               +                                        +----------------------+--------------------------------------+                                      +
|               |                                        | git_modifiers        | xxxxxxx                              |                                      |
+               +                                        +----------------------+--------------------------------------+                                      +
|               |                                        | git_last_modified_by | xxxxxxxxxxxxxxxxx                    |                                      |
+               +                                        +----------------------+--------------------------------------+                                      +
|               |                                        | git_last_modified_at | xxxxxxxxxxxxxxxxxxx                  |                                      |
+               +                                        +----------------------+--------------------------------------+                                      +
|               |                                        | git_file             | smoke/main.tf                        |                                      |
+               +----------------------------------------+----------------------+--------------------------------------+--------------------------------------+
```

**To Reproduce**
`yor tag -d . --skip-tags git_commit, git_file, git_last_modified_at, git_last_modified_by, git_modifiers, git_org, git_repo`

**Expected behavior**

`smoke/main.tf` should have only the `yor_trace` tag added.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: GH actions ubuntu 22.04
 - Yor Version: 0.1.150

**Additional context**
Add any other context about the problem here (e.g. code snippets).

**Comment by jasonckeating** [comment]
Same issue here.

Similarly, running `yor tag --directory terraform/ --tags yor_trace` still adds all tags.

**Comment by rotemavni** [comment]
Hello @jasonckeating, thank you for bringing this into our attention.
We did have an issue with the implementation of the `--tags`, which will be resolved in https://github.com/bridgecrewio/yor/pull/314.

In this PR I also added a test to verify that `--skip-tags` works as expected, given @Octogonapus issue. I believe the issue was resolved in recent versions.
Are you still experiencing the `--skip-tags`  bug?
